features/install.feature

```diff
@@ -78,6 +78,29 @@ Feature: Installation
       And the file "config/initializers/localeapp.rb" should contain "config.api_key = ENV['LOCALEAPP_API_KEY']"
       And the exit status should be 0
 
+  Scenario: Running install with .env
+    In order to configure my non rails project and check my api key is correct
```

Comment:
Should have empty line before and after.

Please describe the corresponding scenario, or remove it (this one
seems a copy paste of another scenario description in the same
feature).